src/pages/public_registration/PublicGeneralContent.tsx

```diff
@@ -91,6 +94,32 @@ export const PublicGeneralContent = ({ registration }: PublicRegistrationContent
   const scopusIdentifier = registration.additionalIdentifiers?.find((identifier) => identifier.sourceName === 'Scopus')
     ?.value;
 
+  const getContextPublisherIdentifierFromId = (id: string | null) => {
+    if (id !== null) {
+      const publisherId = getIdentifierFromId(id);
+      return publisherId;
+    }
+  };
+
+  const publisherIdentifier = getContextPublisherIdentifierFromId((publicationContext as ChapterPublicationContext).id);
```

Comment:
Tror ikke vi trenger en egen funksjon for dette. i stedet kan vi sjekke om vi har en id hvor vi vil finne identifier. Om ikke trenger vi ikke kalle noe. pseudokode blir noe sånt

```suggestion
  const publisherIdentifier = id ? getIdentifierFromId(id) : null;
```

Så må `id` finnes først da